wishlist/wishlist.css

```diff
@@ -0,0 +1,20 @@
+.ecom-nav {
+    height: 8rem;
+}
+
+.main-heading {
+    font-size: 4rem;
+    margin: 2rem;
+    text-align: center;
+    font-weight: bold;
+}
+
+.card-container {
+    padding: 0rem 5rem;
+    justify-content: center;
+}
+
+.overlay-btn-heart {
+    background-color: var(--color-error);
+    color: var(--color-white);
+}
```

Comment:
add new line at the end of code file
Refer here: https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log